hub/hub_nmdc.go

```diff
@@ -229,6 +229,38 @@ func (h *Hub) nmdcAccept(peer *nmdcPeer) error {
 	if err != nil {
 		return err
 	}
+
+	registeredUser := true
+	if registeredUser {
+		err = c.WriteMsg(&nmdc.GetPass{})
+		if err != nil {
+			return err
+		}
+		err = c.Flush()
+		if err != nil {
+			return err
+		}
+		var pass nmdc.MyPass
+		err = c.ReadMsgTo(time.Now().Add(time.Second*60), &pass)
+		if err != nil {
+			return fmt.Errorf("expected password got: %v", err)
+		}
+
+		userPass := "TestPass"
```

Comment:
```golang
ok, err := h.checkUserPass(peer.Name(), string(pass.String))
```
And let the `checkUserPass` function to assume that the password is valid if it matches the name, for example (`name == pass`).